topics/vtex/index.md

```diff
@@ -0,0 +1,14 @@
+---
+related: vtex-commerce,vtex-ecommerce,vtex-io,vtex-store
+created_by: Mariano Gomide de Faria, Geraldo Thomaz
+display_name: Vtex
+logo: vtex.png
+github_url: https://github.com/vtex
+released: 1999
+short_description: VTEX is a enterprise digital Commerce Platform (SaaS/PaaS Cloud based).
+topic: vtex
+url: https://www.vtex.com
+---
+VTEX is the Enterprise Digital Commerce Platform for brands and retailers looking to build a future-proof digital commerce strategy. Powered by a modern platform and a team of digital commerce experts, we enable enterprises to achieve faster time-to-market, reach their customers across any channel and uncover new areas of growth.
+
+Major global brands including Sony, Walmart, Whirlpool, Coca-Cola, Stanley Black & Decker and Nestlé, plus 3,400 active online stores in 38 countries, trust VTEX to accelerate and transform their commerce business.
```

Comment:
unfortunately this is mostly marketing copy, can we pare it down to the tech facts only?